ion-c-sys/src/lib.rs

```diff
@@ -163,7 +182,42 @@ impl ION_STRING {
     }
 }
 
-pub mod result;
+/// Generates easier to use constants for `ION_TYPE`
+/// These exist as CPP macros that don't get translated over from `bindgen`.
+///
+/// Using `ion_types!(NULL)` will generate a constant of the form:
+/// ```
+/// # use ion_c_sys::*;
+/// pub const ION_TYPE_NULL: *mut ion_type = tid_NULL_INT as *mut ion_type;
+/// ```
+macro_rules! ion_types {
+    ( $($name:ident),* ) => {
+        $(
+            paste! {
+                pub const [<ION_TYPE_ $name:upper>]: *mut ion_type =
+                    [<tid_ $name _INT>] as *mut ion_type;
+            }
+        )*
+    };
+}
+
+ion_types!(
+    none,
```

Comment:
This list is missing BOOL (`tid_bool_INT`). Did the PR deal with BOOL and I missed it?

Reply:
Good catch (don't have enough tests obviously)--need to get to #36.